client/my-sites/plan-features-2023-grid/components/popular-badge.tsx

```diff
@@ -0,0 +1,29 @@
+import { getPlanClass } from '@automattic/calypso-products';
+import classNames from 'classnames';
+import PlanPill from 'calypso/components/plans/plan-pill';
+import useFeaturedLabel from '../hooks/use-featured-label';
+
+const PopularBadge: React.FunctionComponent< {
```

Comment:
This reduces some duplication, keeping things more DRY e.g. the featured labels via the use of a hook. As it stands, it's not complete - some styles should live with their component. Also subject to removal, so remains to be seen.